### PR TITLE
feature profile

### PR DESCRIPTION
**Pull request by drumslayert**

### CODE REVIEW

**Comment by karjac14** [comment]
great work! css master! :+1:

**Comment by rosalsm** [comment]
well done :+1:

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
-
+        <link rel="stylesheet" href="octicons/octicons.css"
```
Comment (kellymurray) [inline src/index.html:12]: You can link the Octicons CDN here and not upload all the font files, etc.

File: src/scss/_custom.scss [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+        display: block;
+        font-size: 16px;
+        color: grey;
+        margin-left: 20px;
+
+      }
+      .details {
+        font-size: 14px;
+        list-style: none;
+        padding:1px;
```
Comment (kellymurray) [inline src/scss/_custom.scss:52]: Why `padding` and `padding-top`?

**Pull request merged**